lms/static/scripts/frontend_apps/services/grading.ts

```diff
@@ -6,6 +6,9 @@ export type Student = {
 
   /** API URL for posting outcome results. */
   LISOutcomeServiceUrl: string;
+
+  /** LTI user id for this student. */
+  lmsId: string;
```

Comment:
Rename this `ltiUserId` perhaps, if it matches that field from the LTI launch payload.